en/sql/schema/view_stmt.rst

```diff
@@ -294,15 +294,55 @@ You can drop a query defined in the query specification by using the **DROP QUER
 COMMENT Clause
 --------------
 
-You can change a view's comment with **COMMENT** clause of **ALTER VIEW** syntax.
+View and column comments can be changed by using the **COMMENT** clause of the **ALTER VIEW** syntax.
 
 ::
 
-    ALTER [VIEW | VCLASS] view_name COMMENT [=] 'view_comment';
+    ALTER [VIEW | VCLASS] view_name
+    COMMENT [=] 'view_comment';
+    COMMENT ON {COLUMN | CLASS ATTRIBUTE} <column_comment_definition> [, <column_comment_definition>] ;
+
+        <column_comment_definition> ::= column_name [=] 'column_comment_string'
+
+*   *view_name*: specifies the name of a view to be modified.
+*   *column_name*: specifies the name of a column to be modified.
+*   *view_comment_string*: specifies a view's comment.
```

Comment:
'view_comment_string' is not exists

Reply:
Thank you. The error was corrected by changing 'view_comment' to 'view_comment_string' in BNF.